spec/models/metric/capture_spec.rb

```diff
@@ -10,46 +10,40 @@
                      :capture_threshold_with_alerts => {:vm => capture_rt, :host => capture_rt}
                     }
                   }
-      stub_server_configuration(settings)
+      stub_settings(settings)
     end
 
-    it "realtime vm uses capture_threshold_with_alerts minutes ago" do
+    it "knows realtime vm capture value" do
```

Comment:
One other concern I have about these specs was that previously, we called a single method on various targets and it figured out the right methods to call on the target.  Now, there are two separate methods, so the logic for which methods to call is not around.  For example, there is no test for "a vm with alerts" + "calling standard_capture_threshold".  The logic for "which method to choose" has essentially been extracted, but is there a test for that? (Hope that was clear 😕  )